love/src/components/OLE/NonExposure/NonExposureEdit.jsx

```diff
@@ -65,9 +65,9 @@ export default class NonExposureEdit extends Component {
     this.id = lodash.uniqueId('nonexposure-edit-');
     const logEdit = props.logEdit ?? NonExposureEdit.defaultProps.logEdit;
 
-    logEdit.jiraurl = getLinkJira(logEdit.urls);
-    logEdit.fileurl = getFileURL(logEdit.urls);
-    logEdit.filename = getFilename(getFileURL(logEdit.urls));
+    // logEdit.jiraurl = getLinkJira(logEdit.urls);
```

Comment:
If this code is not going to be used, it should be removed.

Reply:
thanks for catching that!